Dockerfile

```diff
@@ -0,0 +1,15 @@
+# syntax=docker/dockerfile:1
+
+FROM python:2.7-slim-buster
+
+WORKDIR /
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY main.py .env seq.yml util.py /
+
+ENV HTTP_PROXY="http://proxy.net.addenbrookes.nhs.uk:8080"
```

Comment:
can we also not put these in git, not sure if they're classed as sensitive or not, and also if they change it means updating the code.

These can just be passed on running the docker image like this: `docker run <image-id> --env HTTP_PROXY={http-proxy-address} --env HTTPS_PROXY={https-proxy-address}`, or you can pass a file of environment variables with `docker run <image-id> --env-file {config-file}`.

and also add this to the readme.